prog/vm/github_runner.rb

```diff
@@ -145,21 +147,21 @@ def update_billing_record
     end
   end
 
-  def vm
-    @vm ||= github_runner.vm
```

Comment:
The `def_delegators` approach is slightly slower as it does no caching, but I don't think the performance differences will be significant.

Reply:
Since we delegate `vm` to `github_runner.vm`, doesn’t it get cached on `github_runner.vm` instead of `@vm`? If I'm not missing anything, I tried it, and it only runs a DB query the first time, then uses the cached value directly.

Reply:
There is caching in the Sequel association, but it requires more indirection so it will be slower. However, as I mentioned, I don't think the performance differences will be significant, so it's fine to use this approach.